staking-pool-factory/README.md

```diff
@@ -61,3 +61,18 @@ pub fn on_staking_pool_create(
     predecessor_account_id: AccountId,
 ) -> PromiseOrValue<bool>;
 ```
+
+### Usage
+
+Contract initialization:
+```bash
+near call <staking_pool_factory> new '{"staking_pool_whitelist_account_id": <whitelist_contract>}'
+```
+where `<whitelist_contract>` it the account id of the whitelist contract that this factory
+is associated with.
```

Comment:
I think this should be inlined with the deployment, but it's okay to have it as an initialization example with the note of caution.